codecs/wp2/enc/wp2_enc.js

```diff
@@ -576,7 +576,7 @@ var wp2_enc = (function () {
       },
     });
     var ob = {
-      p: function (a, b, c, d) {
+      t: function (a, b, c, d) {
```

Comment:
This is probably unrelated to your PR, but why is the glue not minified?

Reply:
I've fixed this on dev. Needed a prettier ignore file.